babel.config.js

```diff
@@ -1,10 +1,15 @@
 // @flow strict
+
+const tranformPlugin /*: string */ =
```

Comment:
Uses flow comment types: https://flow.org/en/docs/types/comments/

We use babel to strip the flow types but don't use babel on the babel config file itself.